app/controllers/staff/reports_state_controller.rb

```diff
@@ -3,18 +3,26 @@
 class Staff::ReportsStateController < Staff::BaseController
   include Secured
 
+  STATE_TO_POLICY = {
```

Comment:
The policies here could have the `?` be part of the policy name, as in `active: :activate?`, rather than appending the `?` inside the controller methods.